core/assets/bundles/bundle_fr.properties

```
@@ -1980,14 +1980,14 @@ block.spore-press.description = Compresse des spores pour synthétiser du pétro
 block.pulverizer.description = Écrase la ferraille pour en faire du sable. Utile quand il y a un manque de sable naturel.
 block.coal-centrifuge.description = Solidifie le pétrole en blocs de charbon.
 block.incinerator.description = Incinère ou vaporise n'importe quel objet ou liquide qu'il reçoit.
-block.power-void.description = Absorbe toute l'énergie qui va à l'intérieur. Bac à sable uniquement.
-block.power-source.description = Produit de l'énergie à l'infini. Bac à sable uniquement.
-block.item-source.description = Produit des objets à l'infini. Bac à sable uniquement.
-block.item-void.description = Détruit les objets introduits. Bac à sable uniquement.
-block.liquid-source.description = Source de liquide infinie. Bac à sable uniquement.
-block.liquid-void.description = Détruit les liquides introduits. Bac à sable uniquement.
-block.payload-source.description = Produit des charges utiles à l'infini. Bac à sable uniquement.
-block.payload-void.description = Détruit toutes les charges utiles. Bac à sable uniquement.
+block.power-void.description = Absorbe toute l'énergie qui va à l'intérieur. (Bac à sable uniquement)
+block.power-source.description = Produit de l'énergie à l'infini. (Bac à sable uniquement)
+block.item-source.description = Produit des objets à l'infini. (Bac à sable uniquement)
+block.item-void.description = Détruit les objets introduits. (Bac à sable uniquement)
+block.liquid-source.description = Source de liquide infinie. (Bac à sable uniquement)
+block.liquid-void.description = Détruit les liquides introduits. (Bac à sable uniquement)
+block.payload-source.description = Produit des charges utiles à l'infini. (Bac à sable uniquement)
+block.payload-void.description = Détruit toutes les charges utiles. (Bac à sable uniquement)
```

```suggestion
block.power-void.description = Absorbe toute l'énergie qui va à l'intérieur. Bac à sable uniquement.
block.power-source.description = Produit de l'énergie à l'infini. Bac à sable uniquement.
block.item-source.description = Produit des objets à l'infini. Bac à sable uniquement.
block.item-void.description = Détruit les objets introduits. Bac à sable uniquement.
block.liquid-source.description = Source de liquide infinie. Bac à sable uniquement.
block.liquid-void.description = Détruit les liquides introduits. Bac à sable uniquement.
block.payload-source.description = Produit des charges utiles à l'infini. Bac à sable uniquement.
block.payload-void.description = Détruit toutes les charges utiles. Bac à sable uniquement.
```
Non, aucun autre traduction ne le fait et c pas dans le bundle original.
https://github.com/phinner/mindustry/blob/e4253bcbf8fdc638cc52247ac553b5ed697447d4/core/assets/bundles/bundle.properties#L1983-L1990